packages/@ember/-internals/glimmer/tests/integration/helpers/unique-id-test.js

```diff
@@ -1,8 +1,34 @@
-import { RenderingTestCase, strip, moduleFor, runTask } from 'internal-test-helpers';
+import {
+  AbstractTestCase,
+  RenderingTestCase,
+  strip,
+  moduleFor,
+  runTask,
+} from 'internal-test-helpers';
 import { setProperties } from '@ember/object';
+import { uniqueId } from '@ember/helper';
 import { EMBER_UNIQUE_ID_HELPER } from '@ember/canary-features';
 
 if (EMBER_UNIQUE_ID_HELPER) {
+  moduleFor(
+    'Helpers test: {{unique-id}} JS',
+    class JSRenderingTest extends AbstractTestCase {
```

Comment:
I am not sure if the extends AbstractTestCase is required here so semi blindly following patterns here.

Reply:
This seems 👍🏼